app/actions/local/permalink.ts

```diff
@@ -57,7 +57,7 @@ export const showPermalink = async (serverUrl: string, teamName: string, postId:
         const screen = 'Permalink';
         const passProps = {
             isPermalink: openAsPermalink,
-            teamName,
+            teamName: team.name,
```

Comment:
this change is fine, but the entire action should be move the remote actions as there is a potential fetch call here.

Reply:
Also about this action, I wonder what happens if the post being opened belongs to a channel in the same or other team that you are not a member of, it seems that is not handling that.

Perhaps for this PR which is totally about recent mentions is fine, but it definitely needs to be extended to cover the  case I'm mentioning